pubspec.yaml

```
@@ -7,18 +7,31 @@
   sdk: ^3.5.0
 
 dependencies:
+  bloc: ^8.1.4
+  equatable: ^2.0.5
+
   bloc: ^9.0.0
```

YAMLlint (1.35.1)

[error] duplication of key "bloc" in mapping

(key-duplicates)